lang/csharp/src/apache/main/Reflect/ArrayHelper.cs

```diff
@@ -0,0 +1,98 @@
+/*  Copyright 2019 Pitney Bowes Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
[AVRO-2404](https://issues.apache.org/jira/browse/AVRO-2404) (#529) went thru and updated all these to `https://`. I'll insert suggestions that you can apply for each instance.

```suggestion
 *     https://www.apache.org/licenses/LICENSE-2.0
```